linux/roles/tenable/tasks/main.yml

```
@@ -1,39 +1,16 @@
 ---
 
-- name: Gather package facts
-  ansible.builtin.package_facts:
+- name: Stop Nessus Agent service
+  ansible.builtin.service:
+    name: nessusagent
+    state: stopped
```

I'll be curious to see what this does on subsequent runs, after Tenable is uninstalled. Will it fail loudly because `nessusagent` doesn't exist to be stopped, or will it just quietly mark itself as "ok" because the service is confirmed to not be running? Something to keep an eye on.